content/blog/announcing_kubeflow_0.3.md

```diff
@@ -0,0 +1,156 @@
+```
+
+The new tool allows Kubernetes resources and non-K8s resources (e.g. clusters, filesystems, etc...) to be created and deployed consistently. We've divided the deployment process into two steps: 1) creating declarative configs describing your deployment and 2) applying these configs. This makes it easy to customize your deployment and check it into source control for repeatable deployments.
+
+**More options for local development**
+
+ * Minikube deployment now provides a single command shell script based deployment.
+
+ * In addition to minikube, you can now use MicroK8s to easily run Kubeflow on your laptop.
+
+### Develop
+
+In order to deliver a complete platform for building ML products, we are continuing to add and improve the services included in Kubeflow.
+
+**Inference**
+
+We are pleased to announce a number of improvements to our inference capabilities.
+
+Using Apache Beam it is now possible to do batch inference with GPUs (but non distributed) for [TensorFlow](https://www.kubeflow.org/docs/guides/components/tfbatchpredict/).
+
+We are continuing our efforts to make it easy to run TFServing in production by adding a Liveness probe and using fluentd to log r[equest and responses ](https://www.kubeflow.org/docs/guides/components/tfserving/#request-logging)to enable[ ](https://www.kubeflow.org/docs/guides/components/tfserving/#request-logging)model retraining.
+
+In collaboration with NVIDIA we are really excited to be offering more options for online prediction using GPUs by taking advantage of [NVIDIA's TensorRT Inference Server.](https://www.kubeflow.org/blog/nvidia_tensorrt/)
+
+**Hyperparameter tuning**
+
+Finding optimal hyperparameters is one of the biggest barriers to producing high quality models.
+
+In order to make it easy to use hyperparameter tuning without writing any code, we are introducing a new K8s custom controller, StudyJob, that allows a hyperparameter search to be defined using YAML. This [manifest](https://github.com/kubeflow/katib/blob/master/examples/hypb-example.yaml) shows how easy it is to apply hyperband tuning to your jobs.
+
+**Training**
+
+We continue to expand the list of supported frameworks by offering an apha release of a K8s custom controller for Chainer([docs](https://www.kubeflow.org/docs/guides/components/chainer/)).
+
+For PyTorch we're introducing a v1alpha2 for the API that brings parity and consistency with our TFJob operator.
+
+For PyTorch and TFJob we've added a number of highly requested features needed to handle production workloads.
+
+We've added support for gang-scheduling using Kube Arbitrator ([docs](https://www.kubeflow.org/docs/guides/job-scheduling/)) to avoid stranding resources and deadlockings in clusters under heavy load. We've added a TTL to allow garbage collecting old, completed jobs to avoid taxing the K8s API server. To support multi-tenant clusters, the operators can now be scoped to only claim resources in a specified namespace.
+

```

Comment:
Lines 87-93: Remove empty lines in between.

Reply:
Done